dialog/base.go

```diff
@@ -104,6 +104,7 @@ func (d *dialog) Show() {
 		d.win.Resize(d.desiredSize)
 	}
 	d.win.Show()
+	d.Refresh()
```

Comment:
Refresh would surely be better placed before show to avoid possible flicker?

Reply:
Agreed. I will change it.

Reply:
done